### DISCUSSION TITLE
Strange Errors in logs.

### DISCUSSION THREAD

**Discussion by zbindenren** [nats-io/nats-server#4711] [Q&A]
We get sporadically the following errors in the logs:
![image](https://github.com/nats-io/nats-server/assets/1607088/6a944b47-25f6-46ab-9fdd-d0d15ba68be7)

After we restart the nats servers every thing is ok. 

Our certificate provider does unfortunatelly not allow IP addresses in the certificate.

Any Ideas what this might be.

We use nats version 2.9.17 with jetstream.

**Comment by derekcollison** [comment]
Do you have something else trying to connect to those ports? Maybe health checks or something?

**Comment by zbindenren** [comment]
I don't think so. After we restart all nats servers sequentially all is ok.

And yesterday the error started when I restarted one server. I had to restart all servers again and all was ok.

I try to find a reproducer...
